Backend/mentii/book_ctrl.py

```diff
@@ -0,0 +1,42 @@
+import uuid
+from utils.ResponseCreation import ControllerResponse
+from utils import db_utils as dbUtils
+from flask import g
+import utils.MentiiLogging as MentiiLogging
+
+def createBook(bookData, dynamoDBInstance, userRole=None):
+  response = ControllerResponse()
+
+  #g will be not be available during testing
+  #userRole will need to be passed to the function
+  if g:
+    userRole = g.authenticatedUser['userRole']
+  #role is confirmed here incase createBook is called from somewhere other
+  #than app.py createBook()
+  if userRole != 'admin':
+    response.addError('Role error', 'Only admins can create books')
+  elif bookData is None:
```

Comment:
Added check that title and description are in data. chapters field is not required as we do not make it required as per our discussion.